tests/App/AppKernel.php

```diff
@@ -56,6 +57,12 @@ public function registerBundles(): iterable
             new TwigBundle(),
             new DoctrineFixturesBundle(),
         ];
+
+        if (class_exists(StimulusBundle::class)) {
```

Comment:
needed due to https://github.com/sonata-project/SonataAdminBundle/pull/8230/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R60